docs/docs/workflows/config.md

```diff
@@ -0,0 +1,228 @@
+# Configure your cluster
+
+Before your cluster can be created, you need to configure the identity and access management (IAM) for your CSP and choose machine types for your cluster.
```

Comment:
```suggestion
Before you can create your cluster, you need to configure the identity and access management (IAM) for your cloud service provider (CSP) and choose machine types for the nodes.
```
prefer active voice
spell out acronyms on first use on a page (except for those that are more well known than the spelled-out term)